canonical/canonical-design
# Setup dotrun, ci/cd and dev enviroment

## Done

- Allows running the project with dotrun and updates readme
- Add useful scripts to package.json
- Adds CI/CD pipeline

## QA

- Check out the branch and run the project using `dotrun` (if you have run the project previously you may need to `dotrun clean` to refresh your venv)
- Go to  http://localhost:8052 and see that the project runs and with vanilla, JS etc.
- Make a change to HTML, JS and CSS and see that is rebuilds and the changes are visible on refresh
- Go through some of the lint/format scripts in package.json and see that they work
- Check the CI/CD pipeline is passing (and check it _should_ be passing)
